CenterForOpenScience/ember-osf-web
# [ENG-601] Upgrade to Ember 3.15 (Octane)

- Ticket: [ENG-601]
- Feature flag: n/a

## Purpose

Upgrade to latest stable ember-source, ember-data, and ember-cli.

## Summary of Changes

* chore: upgrade to ember-source@~3.15.0
* fix: schedule collections provider active filter push
* chore: upgrade to ember-data@~3.15.0
* fix: underscore relationship names for lookup in relationshipLinks
* chore: upgrade to ember-cli@~3.15.1

## Side Effects

None expected.

## QA Notes

In addition to upgrading Ember, this PR makes a small change to how the active provider filter is added on the Collections Discover page. Should test that discover page results are indeed properly filtered by collection.


[ENG-601]: https://openscience.atlassian.net/browse/ENG-601
